worlds/pokemon_rb/docs/setup_fr.md

```diff
@@ -0,0 +1,102 @@
+# Guide de configuration pour Pokémon Rouge et Bleu : Archipelago
+
+## Important
+
+Comme nous utilisons BizHawk, ce guide s'applique uniquement aux systèmes Windows et Linux.
+
+## Logiciel requis
+
+- BizHawk : [Version BizHawk de TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+   - Les versions 2.3.1 et ultérieures sont prises en charge. La version 2.9.1 est recommandée.
+   - Des instructions d'installation détaillées pour BizHawk peuvent être trouvées sur le lien ci-dessus.
+   - Les utilisateurs Windows doivent d'abord exécuter le programme d'installation des prérequis, qui peut également être trouvé sur le lien ci-dessus.
+- Le client Archipelago intégré, qui peut être installé [ici](https://github.com/ArchipelagoMW/Archipelago/releases)
+   (sélectionnez `Pokemon Client` lors de l'installation).
+- Fichiers ROM Pokémon Rouge et/ou Bleu. La communauté Archipelago ne peut pas les fournir.
+
+## Logiciel en option
+
+-[Pokémon Red and Blue Archipelago Map Tracker](https://github.com/j-imbo/pkmnrb_jim/releases/latest), à utiliser avec [PopTracker](https://github.com/black-sliver/PopTracker/releases)
+
+
+## Configuration de BizHawk
+
+Une fois BizHawk installé, ouvrez EmuHawk et modifiez les paramètres suivants :
+
+- (≤ 2,8) Allez dans Config > Personnaliser. Passez à l'onglet Avancé, puis faites passer le Lua Core de "NLua+KopiLua" à "Lua+LuaInterface". 
+Ensuite redémarrez EmuHawk. Ceci est nécessaire pour que le script Lua fonctionne correctement.
+   **REMARQUE : Même si « Lua+LuaInterface » est déjà sélectionné, basculez entre les deux options et resélectionnez-la. Nouvelles installations**
```

Comment:
This paragraph is really hard to parse. It might be worth to just rewrite it in a different way to have the same message. I think the structure works in english but not at all in french (for instance, starting with "Nouvelles installations" is gramatically odd)